great-idea/css/index.css

```diff
@@ -63,4 +63,130 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+
+
+#container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+#navwrapper {
+	display: flex;
+}
+
+nav {
+	margin-top: 35px;
+	display: flex;
+}
+
+nav > a {
+	display: flex;
+	color: gray;
+	text-decoration: none;
+	font-size: .9em;
+	margin-right: 80px;
+}
+
+.logo {
+	margin-top: 20px;
+}
+
+nav > a:hover {
+	color: royalblue;
+	border-bottom: 1px solid royalblue;
+}
+
+h1 {
```

Comment:
you generally don't want to put with to text type elements, this is usually best left to the font weight and font family.